pkg/cri/resource-manager/requests.go

```diff
@@ -481,6 +491,36 @@ func (m *resmgr) RebalanceContainers() error {
 	return nil
 }
 
+// DeliverPolicyEvent delivers a policy-specific event to the active policy.
+func (m *resmgr) DeliverPolicyEvent(e *events.Policy) error {
+	m.Lock()
+	defer m.Unlock()
+
+	if e.Source == "" {
+		e.Source = "unspecified"
+	}
+
+	m.Info("delivering policy event %s.%s...", e.Source, e.Type)
+
+	method := "DeliverPolicyEvent"
+	changes, err := m.policy.HandleEvent(e)
+
+	if err != nil {
+		m.Error("%s: handling event %s.%s failed: %v", method, e.Source, e.Type, err)
+		return err
+	}
+
+	if changes {
```

Comment:
Does `changes` mean changes to the containers? Or should it be read more like `handled`?

Reply:
Yes, it's 'changes to containers'. But it would probably be better to drop that boolean return altogether. The cache can already tell if there are pending changes to containers.